apps/app.go

```diff
@@ -46,7 +50,7 @@ func (id AppId) String() string {
 }
 
 func (id AppId) ConsulServiceName() string {
-	return strings.Replace(strings.Trim(strings.TrimSpace(id.String()), "/"), "/", ".", -1)
+	return strings.Replace(strings.Trim(strings.TrimSpace(id.String()), "/"), "/", CONSUL_NAME_SEPARATOR, -1)
```

Comment:
Passing variable every time we need get consul name is not an option. Then in events we will need global variable. I'd prefer to move this logic to consul object but this will be a big change. Apps and Events will be affected and so this will lead to huge changes in whole codebase.